packages/atomic-angular/package.json

```diff
@@ -31,7 +31,6 @@
     "@angular/cli": "17.3.8",
     "@angular/compiler-cli": "17.3.12",
     "@coveo/headless": "2.80.0",
-    "@types/jasmine": "5.1.4",
```

Comment:
The jest types and jasmine types get confused in the monorepo, causing all sorts of weird issues.

For instance, when I removed the TestUtil import [here](https://github.com/coveo/ui-kit/pull/4329/files#diff-382cfcbbf7a331ff09270b0a6616192d3df7e64c4043d5d2ec990c508d716b9bL1) I started getting type errors because some of the jest globals were no longer being resolved correctly.

We don't use jasmine at all in our tests, so removing it makes sense and fixes the issues.